content/docs/1.18/opentelemetry.md

```diff
@@ -0,0 +1,115 @@
+{{< /warning >}}
+
+The Jaeger OpenTelemetry backend components are published as Docker images:
+
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/)
+**jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/)
+**all-in-one**        | [hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/](https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/)
+
+## Compatibility
+
+The Jaeger OpenTelemetry components are **almost** backward compatible legacy Jaeger components.
+
+The differences are:
+
+* Health check port changed to `13133`
+* Not all legacy Jaeger flags are exposed (e.g. health check port)
+* Exposed metrics
+
+## Configuration
+
+Jaeger OpenTelemetry components can be configured by a subset of Jaeger legacy flags (or other [configuration sources](../cli/))
+and [OpenTelemetry configuration file](https://opentelemetry.io/docs/collector/configuration/).
+The OpenTelemetry configuration takes precedence over Jaeger configuration.
+The Jaeger OpenTelemetry components use hardcoded default configuration that enables predefined set of components - Jaeger receiver, attribute processor, (storage) exporter.
+The opinionated default configuration ensures compatibility between Jaeger legacy components.
+The user provided OpenTelemetry configuration is merged with the default configuration.
+
+Let's have a look at example configuration:
+
+```sh
+$ docker run --rm -it -v ${PWD}:/config \
+    -e SPAN_STORAGE_TYPE=elasticsearch \
+    jaegertracing/jaeger-opentelemetry-collector \
+    --config-file=/config/config.yaml \
+    --es.server.urls=http://localhost:9200 \
+    --es.num-shards=3
+```
+
+The content of `config.yaml`:
+
+```yaml
+exporters:
+  jaeger_elasticsearch:
+    es:
+      server-urls: http://elasticsearch:9200
+      num-replicas: 2
+processors:
+  attributes:
+    actions:
+      - key: user
+        action: delete
+service:
+  pipelines:
+    traces:
+      processors: [attributes]
+```
+
+* Enables Jaeger receiver (by default) with collector endpoints - gRPC, TChannel, HTTP.
+* Enables Elasticsearch backend with URL http://elasticsearch:9200, 3 primary shards (default is 5) and 2 replica shards (default 1). The CLI flag `--es.server.urls=http://localhost:9200` is superseded by the value from the configuration file.
+* Enables attribute processor (disabled by default). Note that new components have to be explicitly added to the pipeline.
+* Enables health check extension (by default).
```

Comment:
Should it say somewhere that if additional exporters or receivers are added, then the default ones have to be included in the list aswell? I think the list is replaced (I believe) to enable the order to be defined?

Reply:
The line above is explaining that. I will add also that the lists are fully replaced.